### DISCUSSION TITLE
multi-select feature, guide function, parameter decimal points

### DISCUSSION THREAD

**Discussion by antirocker** [keiwando/toonsquid#152] [Ideas]
Hi, thank you for making a great app.
I wrote a review on the App Store, but there is a character limit, so I am writing here inevitably.
The app will get better and better over time, but I think it's what I need the most by my standards.

-If you want to create and import assets from outside, you have to import them one by one. It seems like there should be a multi-select feature.
-I think there should be a guide function. When creating keyframe animation, there is no guide, so there are many cases where you need to find the original position of the key and check the value.
-The parameter value is applied up to one decimal place, but in practice, the movement is often larger than expected. It would be nice if it was applied to the second decimal place.

https://user-images.githubusercontent.com/125040388/218112057-38d2ac10-2221-4506-bf3b-b880470ba7f3.mp4

![#3_HK_idle pose demo_gif](https://user-images.githubusercontent.com/125040388/218112159-e4ba6586-c71a-4bd1-9577-8aab8a99d63f.gif)

**Comment by keiwando** [comment]
Thanks for the suggestions!
Would you mind elaborating on the "guide" functionality for keyframes, maybe with an example? Are we talking about on-canvas guides like the snapping guides?
Also, as a clarification regarding the input fields: The value that you enter is stored as-is and is applied while animating up to many more than just one decimal place, even if the input field only displays the value up to one decimal place in the UI. If you are running into a scenario where you feel like the movement is not as precise as it should be, please send me the corresponding `.tsproj` project file to contact@toonsquid.com and I will take a look to see what might be going on.

**Comment by antirocker** [comment]
thanks for the reply -As for the guide function, it would be nice to have a
snap function, but I suggested it because I thought it would be helpful
just to show the original position or the movement line during animation. -If
it is possible to process more than two decimal places inside the app, I
don't think there is any particular need for adjustment. I'll try animating
the small objects again later and report any oddities.
